### PR TITLE
docs: add release notes for dotnet, java, python

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: docs/src/release-notes-csharp.md [hunk 1]
```diff
@@ -5,6 +5,25 @@ title: "Release notes"
...
+
+|          | Ubuntu 18.04 | Ubuntu 20.04 | Ubuntu 22.04 | Debian 11
+| :--- | :---: | :---: | :---: | :---: | 
+| Chromium | ✅ | ✅ | ✅ | ✅ |
+| WebKit | ✅ | ✅ | ✅ | ✅ |
+| Firefox | ✅ | ✅ | ✅ | ✅ |
+
+### New introduction docs
+
+We're rewrote our docus to be more end-to-end testing focused, see on [playwright.dev](https://playwright.dev/dotnet/docs/intro).
```
Comment (rwoll) [inline docs/src/release-notes-csharp.md:25]:
```suggestion
We rewrote our docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/dotnet/docs/intro).
```

File: docs/src/release-notes-python.md [hunk 2]
```diff
@@ -5,6 +5,25 @@ title: "Release notes"
...
+
+|          | Ubuntu 18.04 | Ubuntu 20.04 | Ubuntu 22.04 | Debian 11
+| :--- | :---: | :---: | :---: | :---: | 
+| Chromium | ✅ | ✅ | ✅ | ✅ |
+| WebKit | ✅ | ✅ | ✅ | ✅ |
+| Firefox | ✅ | ✅ | ✅ | ✅ |
+
+### New introduction docs
+
+We're rewrote our docus to be more end-to-end testing focused, see on [playwright.dev](https://playwright.dev/python/docs/intro).
```
Comment (rwoll) [inline docs/src/release-notes-python.md:25]:
```suggestion
We rewrote our docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/python/docs/intro).
```

**Review by rwoll — CHANGES_REQUESTED** [review]

**Review by aslushnikov — APPROVED** [review]

File: docs/src/release-notes-csharp.md [hunk 3]
```diff
@@ -5,6 +5,25 @@ title: "Release notes"
...
+
+|          | Ubuntu 18.04 | Ubuntu 20.04 | Ubuntu 22.04 | Debian 11
+| :--- | :---: | :---: | :---: | :---: | 
+| Chromium | ✅ | ✅ | ✅ | ✅ |
+| WebKit | ✅ | ✅ | ✅ | ✅ |
+| Firefox | ✅ | ✅ | ✅ | ✅ |
+
+### New introduction docs
+
+We rewrote our docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/dotnet/docs/intro).
```
Comment (rwoll) [inline docs/src/release-notes-csharp.md:25]:
```suggestion
We rewrote our Getting Started docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/dotnet/docs/intro).
```

File: docs/src/release-notes-python.md [hunk 4]
```diff
@@ -5,6 +5,25 @@ title: "Release notes"
...
+
+|          | Ubuntu 18.04 | Ubuntu 20.04 | Ubuntu 22.04 | Debian 11
+| :--- | :---: | :---: | :---: | :---: | 
+| Chromium | ✅ | ✅ | ✅ | ✅ |
+| WebKit | ✅ | ✅ | ✅ | ✅ |
+| Firefox | ✅ | ✅ | ✅ | ✅ |
+
+### New introduction docs
+
+We rewrote our docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/python/docs/intro).
```
Comment (rwoll) [inline docs/src/release-notes-python.md:25]:
```suggestion
We rewrote our _Getting Started_ docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/python/docs/intro).
```

**Review by rwoll — CHANGES_REQUESTED** [review]
Some more edits. Let's qualify what changed—soon enough the whole docs will be there, but for now it's just part of our docs.

**Review by rwoll — APPROVED** [review]

**Pull request merged**